tests/test_check_table_equivalence.py

```diff
@@ -119,6 +120,48 @@ def test_check_fails_when_extra_ids_are_in_per_project_table_only(self):
              '  mock_id_3\n'
              'BigQuery SQL:\n' + formatting.indent(MOCK_QUERY)))
 
+    def test_check_trims_list_of_extra_ids_when_the_list_is_large(self):
+        """When the check finds many extra test_ids, we should trim the list.
+
+        If the list of extra test_id values is very long, we should trim it
+        down, remove duplicates, and sort it.
+        """
+        mock_query_result = 'per_month_test_id,per_project_test_id\n'
+
+        # Add the first 5 values out of order.
+        mock_query_result += ',mock_id_03\n'
+        mock_query_result += ',mock_id_00\n'
+        mock_query_result += ',mock_id_02\n'
+        mock_query_result += ',mock_id_04\n'
+        mock_query_result += ',mock_id_01\n'
+
+        # Add 100 more rows with duplicates.
+        for i in range(0, 100):
+            mock_query_result += ',mock_id_%02d\n' % i
+
+        self.query_executor.execute_query.return_value = mock_query_result
+
+        check_result = self.checker.check(constants.PROJECT_ID_NDT, START_TIME,
+                                          END_TIME)
+        self.assertFalse(check_result.success)
+        self.assertMultiLineEqual(
+            check_result.message,
+            ('Check failed: TABLE EQUIVALENCE\n'
+             'test_id values present in per-project table, but NOT present in '
+             'per-month table:\n'
+             '  mock_id_00\n'
```

Comment:
That's a good point. I think the "right" fix is probably something like we do in `test_query_construct` where we treat different number of spaces as identical. But that feels a bit overkill at the moment, so instead I'll add explicit spaces to the `indent` calls in `check_table_equivalence`.

Fixed.